Libraries/LibGfx/ClassicStylePainter.cpp

```diff
@@ -0,0 +1,329 @@
+/*
+ * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
```

Comment:
I didn't add these classes, so feel free to take some credit and add yourself here as well (if you like.) :)

Reply:
Done, thanks :^)